configuration/concepts/registration.csv

```diff
@@ -33,7 +33,8 @@ bf041739-892f-4744-8cb2-d60903284835,,CIEL:162930,Voodoo,Vaudoo,Voodoo,Vaudoo,,,
 0891f4e7-f95b-4014-a908-4911d4e9e337,,CIEL:162934,Pentecostal,Pentecôtiste,Pentecostal,Pentecôtiste,,,Misc,N/A,,,,,,,,,,,
 664a7f0f-a4bd-435c-b6c2-6ee0233a1310,,CIEL:162935,Seventh-day Adventist,Adventiste,Seventh-day Adventist,Adventiste,,,Misc,N/A,,,,,,,,,,,
 b9eaaf1a-be41-4eef-9e12-6b038786fe03,,CIEL:162936,Jehovah's Witness,Témoin de Jehovah,Jehovah's Witness,Témoin de Jehovah,,,Misc,N/A,,,,,,,,,,,
-6a7da305-b5f2-4484-a0ed-ad59f4de8110,,CIEL:162929,Religion,Religion,Religion,Religion,,,Misc,Coded,,Vaudoo;Catholique;Baptiste;Musulman;Pentecôtiste;Témoin de Jehovah;Adventiste,,,,,,,,,
+7d1fc8af-7f80-4281-b31f-f5b3377d1e7d,,CIEL:162937,Other,Autre,Other,Autre,,,Misc,N/A,,,,,,,,,,,

```

Comment:
@nuhuhmutebi This concept already exists [here](https://github.com/mekomsolutions/openmrs-config-haiti/blob/7bd3979d3b34bcdeff41a96c3523aa01111b2e37/configuration/concepts/commons.csv#L62)

Reply:
@nuhuhmutebi could you fix this

Reply:
Done